processing/src/main/java/org/apache/druid/data/input/impl/CsvInputFormat.java

```diff
@@ -82,6 +83,16 @@ public InputEntityReader createReader(InputRowSchema inputRowSchema, InputEntity
     );
   }
 
+  @JsonIgnore
+  @Override
+  public long getWeightedSize(@Nullable CompressionUtils.Format compressionFormat, long size)
+  {
+    if (CompressionUtils.Format.GZ == compressionFormat) {
+      return size * 4L;
```

Comment:
Better to have this be a constant in `CompressionUtils`

Reply:
fixed